infra/src/main/kotlin/fi/fta/geoviite/infra/geocoding/GeocodingDao.kt

```diff
@@ -62,7 +62,7 @@ class GeocodingDao(
                 and :publication_state = any(rl.publication_states)
               left join layout.km_post_publication_view kmp on kmp.track_number_id = tn.official_id
                 and :publication_state = any(kmp.publication_states)
-                and kmp.state != 'DELETED'
```

Comment:
Kun mergeät tämän, niin huomaa että GeocodingDao:on on mainissa tullut toinenkin metodi joka hakee näitä samoja juttuja (aikaleimalla). Siellä pitää tehdä tämä sama muutos.